src/main/flatbuffers/rlbot.fbs

```diff
@@ -185,4 +185,43 @@ table DesiredGameState {
   carSates:[DesiredCarState];
 }
 
-root_type DesiredGameState;
\ No newline at end of file
+root_type DesiredGameState;
+
+
+enum RenderType : byte {
+    DrawLine2D = 1,
+    DrawLine3D = 2,
+    DrawLine2D_3D = 3,
+    DrawRect2D = 4,
+    DrawRect3D = 5,
+    DrawString2D = 6,
+    DrawString3D = 7
+}
+
+table Color {
```

Comment:
This is unlikely to change, so you could probably use a struct. 

> structs are ideal for data structures that will not change, since they use less memory and have faster lookup

Reply:
I left this as a table so that it can be created once and used multiple times,
It appears that is not possible with structs